site/content/docs/5.3/components/card.md

```diff
@@ -423,6 +460,12 @@ Cards include various options for customizing their backgrounds, borders, and co
 
 Set a `background-color` with contrasting foreground `color` with [our `.text-bg-{color}` helpers]({{< docsref "helpers/color-background" >}}). Previously it was required to manually pair your choice of [`.text-{color}`]({{< docsref "/utilities/colors" >}}) and [`.bg-{color}`]({{< docsref "/utilities/background" >}}) utilities for styling, which you still may use if you prefer.
 
+{{< design-callout-alert >}}
+These variants **with a colored background** should not be used because they do not respect the Orange Design System specifications.
```

Comment:
```suggestion
These variants **with a colored background** should not be used because they do not respect the Orange Design System specifications. The background should either be black or white.
```
Values might need an update

Reply:
I propose:
The variants **with a colored background** (except black background) should not be used because they do not respect the Orange Design System specifications.